hack/builder/version.sh

```diff
@@ -1,3 +1,3 @@
 VERSION=30-9.0.3
-# TODO: reenable ppc64le when new builds are available
+# TODO: reenable ppc64le, arm64 when new builds are available
```

Comment:
do you need to add arm64 to ARCHITECTURES below?

Reply:
Yes, then we can have multi-arch builder image.